src/frontend/contexts/PhotoPromiseContext/types.ts

```diff
@@ -1,17 +1,15 @@
 import {LocationObject} from 'expo-location';
+import {Attachment} from '../../sharedTypes';
 
 export const THUMBNAIL_SIZE = 400;
 export const THUMBNAIL_QUALITY = 30;
 export const PREVIEW_SIZE = 1200;
 export const PREVIEW_QUALITY = 30;
 
-export interface SavedPhoto {
-  // id of the photo in the Mapeo database
-  id: string;
-  type?: 'photo';
-  // If an image is to be deleted
+export type SavedPhoto = Omit<Attachment, 'type'> & {
+  type: 'photo';
```

Comment:
tangential and outside the scope of this PR, but maybe the draft photo types should be renamed to `processed_draft` and `unprocessed_draft`, to be consistent with my suggestion above

Reply:
Generally I agree that we should have descriptive name. But the reason I think it makes sense to keep `type = "photo"` is to align with the attachment `type`. The main purpose of this PR was to get rid of the overhead of changing the types from the backend to the fronted. I wonder if there is an alternative way to being descriptive, without changing the `type`

Reply:
ah right, forgot about that note. yeah then I think keep what you have here. my alternative suggestion would've been to rename the typescript type to indicate that it's a saved photo _attachment_, so that `type: photo` makes sense with that, but not worth sweating the details for now